examples/bSplineCurve_example.cpp

```diff
@@ -78,9 +80,43 @@ int main(int argc, char *argv[])
       mergedCurve.merge(&segmentRight);
       gsInfo << "The merged curve: " << mergedCurve << "\n";
       gsWriteParaview( mergedCurve, "mergedCurve", 100);
+
+      // convert it into bezier segments
+      std::vector<gsBSpline<>> bezSegments = mergedCurve.toBezier();
```

Comment:
it would be better to use a gsMultiPatch<> object as return type for the Bezier segmetns

Reply:
Done!